code/datums/trading/ai.dm

```diff
@@ -27,24 +27,66 @@ They sell generic supplies and ask for generic supplies.
 	)
 	possible_wanted_items = list(
 		/obj/item/device/                        = TRADER_SUBTYPES_ONLY,
+		/obj/item/device/kit					 = TRADER_BLACKLIST_ALL, // They're impossible to make or get outside of mining, just annoys traders.
 		/obj/item/device/assembly                = TRADER_BLACKLIST_ALL,
 		/obj/item/device/assembly_holder         = TRADER_BLACKLIST_ALL,
-		/obj/item/device/encryptionkey/syndicate = TRADER_BLACKLIST,
+		/obj/item/device/encryptionkey			 = TRADER_BLACKLIST_ALL, //Why does ai want NT encryption keys?
 		/obj/item/device/radio                   = TRADER_BLACKLIST_ALL,
 		/obj/item/device/pda                     = TRADER_BLACKLIST_SUB,
-		/obj/item/device/uplink                  = TRADER_BLACKLIST
+		/obj/item/device/chameleon				 = TRADER_BLACKLIST, // Why should it want a chameleon projector
+		/obj/item/device/dociler				 = TRADER_BLACKLIST, //Item unobtaineable
+		/obj/item/device/flashlight/drone		 = TRADER_BLACKLIST, // No drone stuff
+		/obj/item/device/camera_bug				 = TRADER_BLACKLIST, // Traitor stuff
+		/obj/item/device/multitool				 = TRADER_BLACKLIST_SUB, // Hacktool, uplink and robo tool ban
```

Comment:
Is easy to acquire

Reply:
![image](https://user-images.githubusercontent.com/6859396/62105710-6ecf3f00-b2a3-11e9-9b8e-58ccd25edfa9.png)
Blacklist_Sub orders the trader to accept the main item [multitools] but not accept it's subtypes [syndicate hacktool, hacktool used by one of the cyborg types and the third one is a multitool that has 25 tc in it and a syndie uplink.] - I don't think we want the ai looking for these rare subtypes and make selling multitools really obnoxious. [Each one has different id, you could print and give it a multitool and it will decline because it's saying it wants a syndicate uplink multitool.

Reply:
Good point